README.md

```diff
@@ -31,16 +31,27 @@ pip install '.[dev]' --index-url https://gitlab.mpcdf.mpg.de/api/v4/projects/218
 Until we have an official pypi NOMAD release with the plugins functionality. Make
 sure to include NOMAD's internal package registry (via `--index-url` in the above command).
 
-
 ### Run the tests
 
-You can the unit testing using the `pytest` package:
+You can run local tests using the `pytest` package:
 
 ```sh
 python -m pytest -sv
 ```
 
-We recommend to install the `coverage` and `coveralls` packages for a more comprehensive output of the testing:
+where the `-s` and `-v` options toggle the output verbosity.
+For interactive debugging, use the `--pdb` flag.
+When debgugging in the VSCode IDE, add the following to the `launch.json` jobs:
```

Comment:
Add this at the end of the README

Reply:
I'll put it in a subsection of `Development` called `Debugging`.